### ISSUE TITLE
Loss function parameters mismatch

### ISSUE THREAD

**Issue by doem97** [weijiawu/Synthetic-to-Real-Unsupervised-Domain-Adaptation-for-Scene-Text-Detection-in-the-Wild#4]
Hi, the loss function in train() and Loss() have a different number of parameters. (5 params vs 6 params)

Besides, the Loss_target() Class is also missing (in train.py). Are you using a different loss lib other than the provided one? Thanks!

**Comment by doem97** [comment]
More specifically, the domain loss part is missing

**Comment by wenyi-li** [comment]
I come up with the same problem. Is there any solution?
